src/organizations/components/OrgProfileTab/SuspendPaidOrgOverlay.tsx

```diff
@@ -68,7 +68,7 @@ const SupportLink = (): JSX.Element => {
   )
 }
 
```

Comment:
Better to call this the org 'suspension' overlay, as the user can't immediately delete orgs through this.